databases/week2/main.sql

```diff
@@ -0,0 +1,80 @@
+use db2Exercise;
+
+
+--Part 1: Working with tasks
+INSERT INTO task (title, description, created, updated, due_date, status_id)
```

Comment:
Consider getting a formatter as it makes things easier to read - it will really start to matter once the queries get big enough